doc/documentation_data_format.md

```diff
@@ -283,7 +220,90 @@ numeric value or `inf` (lower-case) for steady-state measurements.
+
+The observable table has the following columns:
+
+| observableId | observableName | observableFormula | [observableTransformation] | noiseFormula | [noiseDistribution] |
+| --- | --- | --- | --- | --- | --- |
+| [String] | [String] | [String] | ['lin'(default)&#124;'log'&#124;'log10'] |  [String'log'&#124;Number] | ['laplace'&#124;'normal'] |
+| e.g. | | | | | | 
+| relativeTotalProtein1 | Relative abundance of Protein1 | observableParameter1 * (protein1 + phospho_protein1 ) | lin | noiseParameter1 | normal |
+| ... |  ... | ... | ... | ... |
+
+
+### Detailed field description:
+
+* `observableId` [STRING]
+
+  Any identifier which would be a valid identifier in SBML. This is referenced
+  by the `observableId` column in the measurement table.
+
+* [`observableName`] [STRING, OPTIONAL]
+
+  Arbitrary string describing the observable.
+
+* `observableFormula` [STRING]
+
+  Observation function as plain text formula expression.
+  May contain any symbol defined in the SBML model or parameter table. In the
+  simplest case just an SBML species ID or an `AssignmentRule` target.
+
+  May introduce new parameters of the form `observableParameter${n}`, which
+  are overridden by `observableParameters` in the measurement table
+  (see description there).
+
+- `observableTransformation` [STRING, OPTIONAL]
+
+  Transformation of the observable and measurement for computing the objective
+  function. Must be one of `lin`, `log` or `log10`. Defaults to `lin`.
+  The measurements and model outputs are both assumed to be provided in linear
+  space.
+
+* `noiseFormula` [STRING]
+
+  Noise model parameters as plain text formula expression.
+
+  Measurement noise can be specified as a numerical value which will
+  default to a Gaussian noise model if not specified differently in
+  `noiseDistribution` with standard deviation as provided here. In this case,
+  the same standard deviation is assumed for all measurements for the given
+  observable.
+
+  Alternatively, some formula expression can be provided to specify
+  more complex noise models. A noise model which accounts for relative and
+  absolute contributions could, e.g., be defined as
+  ```
+  noiseParameter1_pErk + noiseParameter2_pErk*pErk
+  ```
+  with `noiseParameter1_pErk` denoting the absolute and `noiseParameter2_pErk`
+  the relative contribution for the observable `pErk`. IDs of noise parameters
+  that need to have different values for different measurements have the
+  structure: `noiseParameter${indexOfNoiseParameter}_${observableId}`
+  to facilitate automatic recognition. The specific values or parameters are
+  assigned in the `noiseParameters` field of the *measurement table*
+  (see above). Any parameters named `noiseParameter${1..n}` *must* be
+  overwritten in the measurement table.
+
+- `noiseDistribution` [STRING: 'normal' or 'laplace', OPTIONAL]
+
+  Assumed noise distribution for the given measurement. Only normally or
```

Comment:
well, together with a log observableTransformation, this can also become a log-normal, log-laplace noise model.

Reply:
seems like somebody implementing the log-versions did not update that :thinking: